java/flight/flight-jdbc-driver/src/test/java/org/apache/arrow/driver/jdbc/accessor/impl/text/ArrowFlightJdbcVarCharVectorAccessorTest.java

```diff
@@ -584,19 +584,23 @@ public void testShouldGetTimestampReturnValidDateWithCalendar() throws Exception
 
   private void assertGetBoolean(Text value, boolean expectedResult) throws SQLException {
     when(getter.get(0)).thenReturn(value);
-
     boolean result = accessor.getBoolean();
     collector.checkThat(result, equalTo(expectedResult));
   }
 
+  private void assertGetBooleanForSQLException(Text value) {
+    when(getter.get(0)).thenReturn(value);
+    collector.checkThrows(SQLException.class, () -> accessor.getBoolean());
+  }
```

Comment:
You should do something similiar this in your other PR

Reply:
Also change the name of this method for `assertGetBooleanThrowingSQLException`

Reply:
Ok, thanks.